src/lightgraphs.jl

```diff
@@ -0,0 +1,113 @@
+
+@inline fadj(g::MetaGraph, x...) = fadj(g.graph, x...)
+@inline badj(g::MetaGraph, x...) = badj(g.graph, x...)
+
+eltype(g::MetaGraph) = eltype(g.graph)
+edgetype(g::MetaGraph) = edgetype(g.graph)
+nv(g::MetaGraph) = nv(g.graph)
+vertices(g::MetaGraph) = vertices(g.graph)
+
+ne(g::MetaGraph) = ne(g.graph)
+edges(g::MetaGraph) = edges(g.graph)
+
+has_vertex(g::MetaGraph, x...) = has_vertex(g.graph, x...)
+@inline has_edge(g::MetaGraph, x...) = has_edge(g.graph, x...)
+
+inneighbors(g::MetaGraph, v::Integer) = inneighbors(g.graph, v)
+outneighbors(g::MetaGraph, v::Integer) = fadj(g.graph, v)  # TODO: why not outneighbors?
```

Comment:
I tried to reuse as much as the original code in MetaGraphs and I inherited this. https://github.com/JuliaGraphs/MetaGraphs.jl/blob/d12c16eb17203e3e1cd040d836f8f1adad7b8ad7/src/MetaGraphs.jl#L75.

Reply:
Not sure it is very useful though. Maybe change it for the sake of coherence?
Btw, why are only some of these methods inlined?

Reply:
Sure. Honestly, I can't remember why I inlined them

Reply:
Let's keep it that way for now? When I benchmark we may see how it affects performance

Reply:
Maybe for a future PR, but it might make sense to see if we can get speedups by using `@inbounds` (which might mean we would need several `@propagate_inbounds` annotations)